src/modules/content/content.joi.js

```diff
@@ -19,6 +19,21 @@ const baseUpdateSchema = {
     time  : Joi.string().optional()
 };
 
+const answer = Joi.object({
+    text      : Joi.string().required(),
+    isCorrect : Joi.boolean().required()
+}).required();
+
+const question = Joi.object({
+    question : Joi.string().required(),
+    answers  : Joi.object({
+        a1 : answer,
+        a2 : answer,
+        a3 : answer,
+        a4 : answer,
+        a5 : answer
```

Comment:
hmm i would say may 4 answers should be the norm 
well we need to think about students time